### ISSUE TITLE
Docker image with tag 0.9.0 fails with symbol not found

### ISSUE THREAD

**Issue by erickr** [Jean-Beru/docker-mailcatcher#11]
Thank you for your updated images! However,the last pushed image fails with a symbol not found as seen below:

```
docker run jeanberu/mailcatcher:0.9.0 
Unable to find image 'jeanberu/mailcatcher:0.9.0' locally
0.9.0: Pulling from jeanberu/mailcatcher
Digest: sha256:6505a60750f88e5a2725f97eea4007ed529e506802ef5acbc0044796bb240c6a
Status: Downloaded newer image for jeanberu/mailcatcher:0.9.0
Starting MailCatcher v0.9.0
<internal:/usr/lib/ruby/3.2.0/rubygems/core_ext/kernel_require.rb>:85:in `require': cannot load such file -- sqlite3/sqlite3_native (LoadError)
	from <internal:/usr/lib/ruby/3.2.0/rubygems/core_ext/kernel_require.rb>:85:in `require'
	from /usr/lib/ruby/gems/3.2.0/gems/sqlite3-1.7.1-x86_64-linux/lib/sqlite3.rb:6:in `rescue in <top (required)>'
	from /usr/lib/ruby/gems/3.2.0/gems/sqlite3-1.7.1-x86_64-linux/lib/sqlite3.rb:2:in `<top (required)>'
	from <internal:/usr/lib/ruby/3.2.0/rubygems/core_ext/kernel_require.rb>:85:in `require'
	from <internal:/usr/lib/ruby/3.2.0/rubygems/core_ext/kernel_require.rb>:85:in `require'
	from /usr/lib/ruby/gems/3.2.0/gems/mailcatcher-0.9.0/lib/mail_catcher/mail.rb:6:in `<top (required)>'
	from <internal:/usr/lib/ruby/3.2.0/rubygems/core_ext/kernel_require.rb>:85:in `require'
	from <internal:/usr/lib/ruby/3.2.0/rubygems/core_ext/kernel_require.rb>:85:in `require'
	from /usr/lib/ruby/gems/3.2.0/gems/mailcatcher-0.9.0/lib/mail_catcher/smtp.rb:5:in `<top (required)>'
	from <internal:/usr/lib/ruby/3.2.0/rubygems/core_ext/kernel_require.rb>:85:in `require'
	from <internal:/usr/lib/ruby/3.2.0/rubygems/core_ext/kernel_require.rb>:85:in `require'
	from /usr/lib/ruby/gems/3.2.0/gems/mailcatcher-0.9.0/lib/mail_catcher.rb:183:in `block (2 levels) in run!'
	from /usr/lib/ruby/gems/3.2.0/gems/mailcatcher-0.9.0/lib/mail_catcher.rb:239:in `rescue_port'
	from /usr/lib/ruby/gems/3.2.0/gems/mailcatcher-0.9.0/lib/mail_catcher.rb:182:in `block in run!'
	from /usr/lib/ruby/gems/3.2.0/gems/eventmachine-1.0.9.1/lib/eventmachine.rb:193:in `run_machine'
	from /usr/lib/ruby/gems/3.2.0/gems/eventmachine-1.0.9.1/lib/eventmachine.rb:193:in `run'
	from /usr/lib/ruby/gems/3.2.0/gems/mailcatcher-0.9.0/lib/mail_catcher.rb:180:in `run!'
	from /usr/lib/ruby/gems/3.2.0/gems/mailcatcher-0.9.0/bin/mailcatcher:6:in `<top (required)>'
	from /usr/bin/mailcatcher:25:in `load'
	from /usr/bin/mailcatcher:25:in `<main>'
<internal:/usr/lib/ruby/3.2.0/rubygems/core_ext/kernel_require.rb>:85:in `require': Error relocating /usr/lib/ruby/gems/3.2.0/gems/sqlite3-1.7.1-x86_64-linux/lib/sqlite3/3.2/sqlite3_native.so: posix_fallocate64: symbol not found - /usr/lib/ruby/gems/3.2.0/gems/sqlite3-1.7.1-x86_64-linux/lib/sqlite3/3.2/sqlite3_native.so (LoadError)
	from <internal:/usr/lib/ruby/3.2.0/rubygems/core_ext/kernel_require.rb>:85:in `require'
	from /usr/lib/ruby/gems/3.2.0/gems/sqlite3-1.7.1-x86_64-linux/lib/sqlite3.rb:4:in `<top (required)>'
	from <internal:/usr/lib/ruby/3.2.0/rubygems/core_ext/kernel_require.rb>:85:in `require'
	from <internal:/usr/lib/ruby/3.2.0/rubygems/core_ext/kernel_require.rb>:85:in `require'
	from /usr/lib/ruby/gems/3.2.0/gems/mailcatcher-0.9.0/lib/mail_catcher/mail.rb:6:in `<top (required)>'
	from <internal:/usr/lib/ruby/3.2.0/rubygems/core_ext/kernel_require.rb>:85:in `require'
	from <internal:/usr/lib/ruby/3.2.0/rubygems/core_ext/kernel_require.rb>:85:in `require'
	from /usr/lib/ruby/gems/3.2.0/gems/mailcatcher-0.9.0/lib/mail_catcher/smtp.rb:5:in `<top (required)>'
	from <internal:/usr/lib/ruby/3.2.0/rubygems/core_ext/kernel_require.rb>:85:in `require'
	from <internal:/usr/lib/ruby/3.2.0/rubygems/core_ext/kernel_require.rb>:85:in `require'
	from /usr/lib/ruby/gems/3.2.0/gems/mailcatcher-0.9.0/lib/mail_catcher.rb:183:in `block (2 levels) in run!'
	from /usr/lib/ruby/gems/3.2.0/gems/mailcatcher-0.9.0/lib/mail_catcher.rb:239:in `rescue_port'
	from /usr/lib/ruby/gems/3.2.0/gems/mailcatcher-0.9.0/lib/mail_catcher.rb:182:in `block in run!'
	from /usr/lib/ruby/gems/3.2.0/gems/eventmachine-1.0.9.1/lib/eventmachine.rb:193:in `run_machine'
	from /usr/lib/ruby/gems/3.2.0/gems/eventmachine-1.0.9.1/lib/eventmachine.rb:193:in `run'
	from /usr/lib/ruby/gems/3.2.0/gems/mailcatcher-0.9.0/lib/mail_catcher.rb:180:in `run!'
	from /usr/lib/ruby/gems/3.2.0/gems/mailcatcher-0.9.0/bin/mailcatcher:6:in `<top (required)>'
	from /usr/bin/mailcatcher:25:in `load'
	from /usr/bin/mailcatcher:25:in `<main>'

```

**Comment by MichalSkoula** [comment]
For me, its failing also with 0.8.2

0.8.0 is working well

**Comment by Jean-Beru** [comment]
Thanks for the report!

This error seems to be the same as https://github.com/sparklemotion/sqlite3-ruby/issues/434. I fixed it by using `alpine:3.8`. 

I will push the fixed image in few minutes. Don't hesitate to reopen this issue if it doesn't work.
